.ci/Jenkinsfile

```diff
@@ -9,6 +9,11 @@ pipeline {
     BASE_DIR = "src/github.com/elastic/${env.REPO}"
     JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
     PIPELINE_LOG_LEVEL = 'INFO'
+    DRA_OUTPUT = 'release-manager.out'
+    SLACK_CHANNEL = '#elastic-agent-control-plane'        // TODO: what slack channel
```

Comment:
shipper is data plane so either `elastic-agent-data-plane` or `ingest-notifications` could be fine
@jlind23 wdyt on using `ingest-notifications`?